### PR TITLE
db: fix unicode on mysql

### PR DESCRIPTION
**Pull request by RustyBower**

### CODE REVIEW

**Comment by deathbybandaid** [comment]
After some testing, I can confirm this clears issue #1650

**Comment by RustyBower** [comment]
I did just configure this for a single column in nick_values, but we should chat about all the potentially places people could put unicode characters and configure appropriately.

**Comment by deathbybandaid** [comment]
I would imagine this should be added to channel_values,, as well as the upcoming plugin_values in PR #1621

**Comment by dgw** [comment]
What @deathbybandaid said. Fixing this for only one DB column is no good; we need to allow Unicode pretty much anywhere there's a string/text. Some IRCds do in fact allow Unicode nicknames, for example.

File: sopel/db.py [hunk 1]
```diff
@@ -59,11 +59,13 @@ class NickValues(BASE):
     NickValues SQLAlchemy Class
     """
     __tablename__ = 'nick_values'
+    __table_args__ = {'mysql_engine': 'InnoDB', 'mysql_charset': 'utf8mb4', 'mysql_collate': 'utf8mb4_unicode_ci'}
     nick_id = Column(Integer, ForeignKey('nick_ids.nick_id'), primary_key=True)
     key = Column(String(255), primary_key=True)
     value = Column(String(255))
 
 
+
```
Comment (dgw) [inline sopel/db.py]: Why was this added…?

Comment (RustyBower) [reply sopel/db.py]: great question. fixing it tho lol

**Comment by deathbybandaid** [comment]
Fresh database with mysql seems good with the `__table_args__` setup.
I will try testing with sqlite and postgres later today.

I think those 3 databases are the only ones we should focus on. Any other database flavors can wait until people have issues with them.

**Comment by deathbybandaid** [comment]
Should we be concerned about line length with this?

**Comment by RustyBower** [comment]
Fixed the line length and additional table issues. I'll continue to do some research into defaulting this for the entire table.

**Comment by dgw** [comment]
(And I won't give Rusty time to "fix it anyway", either. :smiling_imp:)

**Comment by Exirel** [comment]
Awesome fix! :+1:
